packages/prosekit/CHANGELOG.md

```
@@ -1,5 +1,13 @@
 # prosekit
 
+## 0.22.0-beta.24
+
+### Minor Changes
+
+- [`2800bb4`](https://github.com/ocavue/prosekit/commit/2800bb4d7ee79cc4f236e9595e1d07570b7789b8) ![](https://prosekit.dev/b/extensions)
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)